testdata/sqllogictests_sqlite/read.slt

```diff
@@ -4,3 +4,26 @@ query I
 SELECT count(*) FROM read_sqlite('${SQLITE_DB_LOCATION}', 'bikeshare_stations');
 ----
 102
+
+query I
+SELECT count(*) FROM read_sqlite('./data/northwind.db', 'Customers');
+----
+93
+
+# TODO: change db location to our own test directory on github once this is merged
+query I
+SELECT COUNT(*) FROM read_sqlite('https://github.com/jpwhite3/northwind-SQLite3/raw/main/dist/northwind.db','Orders');
+----
+16282
+
+query III
+SELECT OrderID, OrderDate, RequiredDate, ShippedDate FROM FROM read_sqlite('https://github.com/jpwhite3/northwind-SQLite3/raw/main/dist/northwind.db','Orders');
```

Comment:
I don't know how I feel about committing a 25mb file to our repo just for these tests. 

I think you can either:
- create a very small database with a subset of this data in the repo and commit it
- upload it to the google bucket with the other (larger) test databases.
- in SLTs create a database with data in the desired forms.